pkg/devicetwin/dtmanager/twin.go

```diff
@@ -88,13 +88,18 @@ func dealTwinSync(context *dtcontext.DTContext, resource string, msg interface{}
 		return nil, errors.New("msg not Message type")
 	}
 	result := []byte("")
-	msgTwin, err := dttype.UnmarshalDeviceTwinUpdate(message.Content.([]byte))
+	content, ok := message.Content.([]byte)
+	if !ok {
+		return nil, errors.New("asseration failed")
```

Comment:
since assertion failed , it indicates the message content is invalid. maybe we can return an error message indicating the same.

Reply:
Modified return message to "invalid message content".